frontend/src/components/AccountSentTable.vue

```diff
@@ -1,12 +1,15 @@
 <template>
   <div>
-    <div class="text-caption q-mb-sm">
-      {{ $t('AccountSentTable.stored-on-device') }}.
-      <router-link
-        class="cursor-pointer hyperlink"
-        :to="{ name: 'FAQ', hash: '#why-cant-I-see-my-send-history-on-different-devices' }"
-        >{{ $t('AccountSentTable.learn-more') }}</router-link
-      >.
+    <div class="flex row justify-between q-mb-sm">
+      <div class="text-caption self-end">
+        {{ $t('AccountSentTable.stored-on-device') }}.
+        <router-link
+          class="cursor-pointer hyperlink"
+          :to="{ name: 'FAQ', hash: '#why-cant-I-see-my-send-history-on-different-devices' }"
+          >{{ $t('AccountSentTable.learn-more') }}</router-link
+        >.
+      </div>
+      <base-button @click="clearHistory" size="md" :label="$t('AccountSent.clear-history')" />
     </div>
```

Comment:
*arbitrary comment location*

We should edit the description of this PR to make sure it mentions, but doesn't close #471, because I think we should still at least consider a future feature that allows users to turn off send history altogether.

Reply:
I split the turn "off send history" into issue #545 in order for us to close 471. Happy to close 545 and leave 471 if that makes more sense.

Reply:
Oh didn't realize that! Sorry I missed it, that definitely works.